<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaDeParada</class>
 <widget class="QDialog" name="TelaDeParada">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>256</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Trebuchet MS</family>
    <pointsize>12</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelTitulo">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Trebuchet MS</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Parada de Máquina</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMaquina">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Máquina:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOP">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ordem de Produção:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDataInicio">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Data Início da Parada:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHorainicio">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Hora Início da Parada:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMotivo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Motivo:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditMaquina">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOP">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxMotivo">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEditDataInicio">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEditHoraInicio">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSalvar">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>170</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Salvar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
